infrastructure/src/app/http.rs

```diff
@@ -186,7 +186,7 @@ mod default {
     use super::*;
 
     pub fn bind_addr() -> String {
-        "::1".to_string()
+        "::".to_string()
```

Comment:
I wouldn't do that, we can override in the configuration. But keeping that to "localhost" instead to "wide open" makes more sense IMHO.

Reply:
Here, this is the setting: https://github.com/trustification/trustification/blob/d652456df89b8790a9fae3049b66f8e48e1f80ff/infrastructure/src/app/http.rs#L133

So just going for `HTTP_SERVER_BIND_ADDR=::` should work.

Reply:
Fixed